torch/testing/_internal/common_utils.py

```diff
@@ -1462,7 +1462,8 @@ def _assertGradAndGradgradChecks(test_case, apply_fn, inputs):
     test_case.assertTrue(gradgradcheck(apply_fn, inputs))
 
 
-dtype2prec = {torch.float: 1e-5,
+# Using @precisionOverride is the recommended way instead of this.
```

Comment:
maybe add a bit more context: using precisionOverride specific to your test is the recommended way of doing this.  These are just some values that worked for test_nn.